src/argilla/client/feedback/integrations/huggingface/dataset.py

```diff
@@ -421,6 +451,11 @@ def from_huggingface(
                         if value is not None:
                             if question.type == QuestionTypes.ranking:
                                 value = [{"rank": r, "value": v} for r, v in zip(value["rank"], value["value"])]
+                            elif question.type == QuestionTypes.span:
+                                value = [
+                                    {"start": s, "end": e, "label": l}
+                                    for s, e, l in zip(value["start"], value["end"], value["label"])
```

Comment:
in terms of human-readability, I also like to add the extracted text normally. Something like`value["text"][value["start"]:value["end"]]` but perhaps this is difficult top map back into the correct format  when calling `from_huggingface`?

Reply:
This change is not hard to add, if it makes sense.